README.md

```diff
@@ -15,35 +15,52 @@ A PHP7 implementation of the [GraphQL specification](http://facebook.github.io/g
 
 When we started developing this project there were two GraphQL implementations available for PHP developers; one from 
 [Webonyx](https://github.com/webonyx/graphql-php/) and one from [Youshido](https://github.com/youshido/graphql/). 
-The one from Webonyx is a direct port of the reference implementation and it includes a lot of things that we think 
-should be provided as separate libraries. The other implementation from Youshido has some neat ideas, but unfortunately 
-it does not fulfill the GraphQL specification. After doing some research, we concluded that neither of these
-implementations met our needs, so we decided to start coding and see what we would come up with.
+The one from Webonyx is a direct port of the reference implementation and it does not use the full potential of 
```

Comment:
"does not utilize the full potential of modern PHP" -> "does not fully use modern PHP"

"After doing some research" -> "After some research"

Reply:
Fixed.